src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -295,25 +295,31 @@ protected void SetBorder(Thickness thickness, Brush brush, CornerRadius cornerRa
 			}
 			else
 			{
-				var borderColor = Colors.Transparent;
-				var borderImage = "";
-
+				var borderWidth = $"{thickness.Top.ToStringInvariant()}px {thickness.Right.ToStringInvariant()}px {thickness.Bottom.ToStringInvariant()}px {thickness.Left.ToStringInvariant()}px";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'thickness' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5417387)